### PR TITLE
feat: save/read from local file

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

**Review by doodlewind — CHANGES_REQUESTED** [review]
I think it's currently a non-goal for us to use WASM-based SQLite in our Web version. We intend to use JSBridge-based PostgresDB (that runs natively in OctoBase) for local persistence in our desktop client.

File: packages/playground/src/components/debug-menu.ts [hunk 1]
```diff
@@ -203,6 +217,38 @@ export class DebugMenu extends LitElement {
...
+    }
+    const input = <SlInput>document.getElementById('input-file-name');
+    const value = input.value;
+    const filename = value.endsWith('.affine') ? value : `${value}.affine`;
+    getFileHandle(filename, this.fsHandle).then(async handle => {
+      if (!handle) {
+        handle = await createFileHandle(filename, this.fsHandle);
+      }
+      if (handle) {
+        await writeContentToFile(handle, Workspace.toLocal(this.workspace));
```
Comment (doodlewind) [inline packages/playground/src/components/debug-menu.ts:244]: Seems that we only serialize YDoc to disk for now?

Comment (himself65) [reply packages/playground/src/components/debug-menu.ts]: yep, im still working on that

Comment (himself65) [reply packages/playground/src/components/debug-menu.ts]: fixed

**Comment by himself65** [comment]
use https://github.com/GoogleChromeLabs/browser-fs-access#readme

**Comment by doodlewind** [comment]
Closing this since we are going to work on file format at https://github.com/toeverything/OctoBase/issues/173. Before the OctoBase implementation, we shouldn't support inner format in BlockSuite that may make migration harder.

**Pull request closed without merging**